pkg/tfbridge/metadata.go

```diff
@@ -61,3 +61,12 @@ func (info *MetadataInfo) assertValid() {
 	contract.Assertf(info.Path != "", "Path must be non-empty")
 
 }
+
+// trim the metadata to just the keys required for the runtime phase
+// in the future this method might also substitute compressed contents within some keys
+func (info *MetadataInfo) ExtractRuntimeMetadata() *MetadataInfo {
+	data, _ := metadata.New(nil)
+	metadata.CloneKey("auto-aliasing", info.Data, data)
```

Comment:
It looks like we don't have an existing var for `"mux"`.
```suggestion
	metadata.CloneKey(aliasMetadataKey, info.Data, data)
```